core/profile/injector.py

```
@@ -0,0 +1,115 @@
+                        "type": tag_type,
+                        "value": value,
+                        "source": PROFILE_SCOPE_PREFIX,
+                        "mtime": mtime,
+                        "stale": age_days > 90,
+                    })
+                except (OSError, UnicodeDecodeError):
+                    logger.warning("[画像注入] 读取失败: %s", filepath, exc_info=True)
+        except OSError:
+            logger.warning("[画像注入] 列出目录失败: %s", profile_dir, exc_info=True)
+
+        # 去重：非 stale 优先，同 mtime 保留后出现的
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)